changelog.json

```
@@ -1,5 +1,21 @@
 {
   "releases": {
+    "2.3.0": [
+      "[New] Prompt users to create a fork if they fail to push to a GitHub repository due to permissions - #8548",
+      "[Added] Show link to upstream pull requests in pull request list of a fork - #8965",
+      "[Added] Show a warning if the user doesn't have write access to push to the repository and offer to make a fork - #8939",
```

This is my bad, sorry about that! I should have linked the issue, not the PR

```suggestion
      "[Added] Show a warning if the user doesn't have write access to push to the repository and offer to make a fork - #8938",
```